docs/software/switch-config.md

```diff
@@ -74,6 +74,16 @@ Wide switch mode also sends current transmitter power to the flight controller t
 
 Ardupilot Modes don't line up very well with the standard -100% (988us) to +100% (2012us) output range in OpenTX when using a 6-position selector as input. Both the first two and the last two positions get binned into Mode 1 and Mode 6 respectively. To get the full 6 Ardupilot modes, go to the Outputs page on the OpenTX model setup and set the min / max for the channels to -75% / +75%.
 
+## Full Resolution Switch Modes :new:
+
+Version 3.0 brought Full Resolution Modes to both Team2400 and Team900. There are three new Switch Modes for these new RF Modes:
+
+1. 8CH Mode - Channels 1 to 4 and Channels 6 to 9 are sent 10-bit, Full Resolution at the current selected Packet Rate, along with Channel 5 (AUX1) in 1-bit (2-position) for Arming.
+
+2. 12CH Mode - Channels 1 to 4 are sent 10-bit, Full Resolution with Channel 5 (AUX1) still in 1-bit, at the selected Packet Rate. Channels 6 to 13 are then sent at 10-bit but at half rate.
+
+3. 16CH Mode - All the Channels are sent 10-bit but at half rate.

```

Comment:
Use the same labels as there are on the screen (and in the same order? not sure about that. The order is weird in the code because of technically how it works and that "Mixed" only makes sense after seeing Rate/2)
`8ch`
`16ch Rate/2`
`12ch Mixed`